docs/testrunner-toolkit/configuration/espresso.md

```diff
@@ -111,6 +111,112 @@ __Example__:
     - "10.0"
 ```
 
+## `devices`
+
+__Description__: Field for defining devices request.
+
+A real device can be selected using two different manner:
+- Specific device selection, through device IDs
+- Filtered device selection, with several contraints such as `name`, `platformVersion`, etc...
+
+When an ID is specified, it is prioritary on other settings.
```

Comment:
If the device is not available, it will wait until it become available. Specifying an `id` involves a strict match on that `id`.
No timelimit that I am aware of. We will introduce a limit at `saucectl` level later, that will just make `saucectl` stop waiting when a specified duration has been reached. The job itself will still be waiting for the device to be available.